docs/helpers.md

```diff
@@ -0,0 +1,61 @@
+# Helper models, views, and ModelAdmins
+
+## OnlineBase
+
+`cms.models.OnlineBase` provides publication controls.
+It provides a single field: `is_online`. When it is `False` (unchecked), it will not appear on the front-end of your website.
+
+You do not have to do anything other than inherit from OnlineBase for this to work; objects that are not online are automatically excluded from querysets via a custom manager, `OnlineBaseManager`.
+We will go into more detail about that later.
+
+`cms.admin.OnlineBaseAdmin` is the companion `ModelAdmin` for this model; derivatives of OnlineBase should probably inherit from this for their user model. It defines `PUBLICATION_FIELDS`, which you can use on your model's admin thusly:
+
+```python
+class YourModelAdmin(OnlineBaseAdmin):
+    fieldsets = [
+        # ... your fieldsets here ...
+        OnlineBaseAdmin.PUBLICATION_FIELDS,
+    ]
+```
+
+`OnlineBaseAdmin` also adds a helpful list `action`: you will be able to turn off items as a batch from the list view.
+
+## PageBase
+
+`cms.models.PageBase` provides the publication control that `OnlineBase` does, but also contains many useful fields that are fairly typical for a typical article-like thing on a website. If an instance of your model lives at its own URL and has a title, such as a news article or a blog post, then you almost certainly want to inherit from `PageBase`. The CMS's own `Page` model inherits from it.
+
+PageBase defines several fields that make this useful for article-like things:
+
+* A title and slug
+* SEO controls: title tag override, meta description, robots controls
+* Fields for Twitter and Facebook (OpenGraph) cards
+
+`cms.admin.PageBaseAdmin` is the companion `ModelAdmin`; at risk of repeating ourselves, anything that derives from `PageBase` definitely wants to use `PageBaseAdmin`. It prepopulates your `title` and `slug` field automatically, and offers the following fieldsets:
+
+* `PUBLICATION_FIELDS`, inherited from `OnlineBase`
+* `SEO_FIELDS` for SEO controls
+* `OPENGRAPH_FIELDS`, to control how OpenGraph card rendering (Facebook and others)
+* `OPENGRAPH_TWITTER_FIELDS`, to control how Twitter cards are rendered
+
+The companion `cms.views.PageDetailView` is a class-based view that takes care of putting `PageBase` fields into the template context so that they can be seen by the CMS's [template functions](template-functions.md). There is also `cms.views.PageDetailMixin`, which does not inherit from Django's `DetailView`.
```

Comment:
You mention that `cms.views.PageDetailMixin` exists but don't go into the functionality that it provides. Even just saying that it provides the same functionality and `cms.views.PageDetailView` but doesn't extend `DetailView` would be good.

Reply:
Good idea.